localnet/with-ibc/docker-compose.yaml

```diff
@@ -0,0 +1,48 @@
+version: "3"
+
+services:
+  finschia-app:
+    container_name: finschia-app
+    image: "finschia/finschianode:1.0.0-dynamiclink2"
+    ports:
+      - "26657:26657"
+      - "1317:1317"
+      - "9090:9090"
+    volumes:
+      - "./finschia/template:/template"
+      - "fnsad_data:/root"
+    entrypoint: /template/run_finschia.sh
+    extra_hosts:
+      - host.docker.internal:host-gateway
+
+  finschia2-app:
+    container_name: finschia2-app
+    image: "finschia/finschianode:1.0.0-dynamiclink2"
+    ports:
+      - "26557:26657"
```

Comment:
Is it okay that `finschia-app` and `finschia2-app` use same Ostracon port?? I think it shouldn't

Reply:
finschia-app uses 26657:26657 and finschia2-app uses 26557:26657
so it is slightly different :)

Reply:
And the script doesn't work for me as well. It was stuck like this.

![Screenshot from 2023-07-07 13-56-43](https://github.com/Finschia/hackathon/assets/41176085/8b70cadd-ea24-4f40-b800-026727a82945)

Reply:
I recommend to use `-p` flag when `docker compose up` to avoid this issue